tests/integration/Core/Repository/Values/User/Limitation/BaseLimitationTest.php

```diff
@@ -6,7 +6,10 @@
  */
```

Comment:
This one is missing strict type declaration too.
```suggestion
 */
 declare(strict_types=1);
 
```

Reply:
Very large part of Core is missing `strict_types`. It is not intention of this PR to adapt already existing classes.

Reply:
If improving the codebase whenever possible (adding one line in this case) is problematic, let's skip it.